doc/analytics.md

```diff
@@ -28,14 +28,20 @@ endpoints or rake tasks.
 
 Here are some example queries to pull out particular insights.
 
-### How many subscribers does a list have at a particular point in time
+### How many subscribers does a list have
 
 ```ruby
+Subscription.active.where(subscriber_list_id: 4194).count
+```
+
+### How many subscribers did a list have at a particular point in time
+
+```ruby
+date = "2018-6-01"
 Subscription.where(
-    "created_at > ? AND (ended_at IS NULL OR ended_at <= ?)",
-    "2018-06-01",
-    "2018-07-01"
-  ).where(subscriber_list_id: 4194).count
+  "created_at < ? AND (ended_at IS NULL OR ended_at <= ?)",
```

Comment:
Should this be OR? Just running it locally and getting smaller numbers that expected